docs/develop/python/message-passing.mdx

```diff
@@ -2,7 +2,7 @@
 id: message-passing
 title: Messages - Python SDK feature guide
 sidebar_label: Messages
-description: Learn how to develop with Signals, Queries, and Updates using the Python SDK in Temporal. Master defining Signal handlers, customizing names, sending Signals, and handling Queries dynamically.
+description: Learn how to develop with Queries, Signals, and Updates using the Temporal Python SDK.
```

Comment:
```suggestion
description: Develop with Queries, Signals, and Updates using the Temporal Python SDK.
```

Should consider putting back some of the meta description here though for SEO.

Reply:
Thanks, done!